onnxruntime/core/providers/op_kernel_type_control.h

```diff
@@ -146,6 +169,16 @@ struct EnabledTypes {
 
 // public macros
 
+/** Data types that are used in DataTypeImpl::AllTensorTypes()
+*/
+#define ORT_OP_KERNEL_TYPE_CTRL_ALL_TENSOR_DATA_TYPES \
+  bool,                                               \
```

Comment:
>   [](start = 0, length = 2)

nit: formatting

Reply:
auto formatted by clang-format.

---
In reply to: [569679793](https://github.com/microsoft/onnxruntime/pull/6547#discussion_r569679793) [](ancestors = 569679793)